spec/features/authentication/show_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'view driver profile', %{
+  As an authenticated user
+  I want to view reviews on driver
+  So that I can find out more about driver's driving skills
+  } do
+  # ACCEPTANCE CRITERIA
+  # * I can visit driver's profile
+  # * I can read all the previous reviews in descending order
+
+  let!(:driver) { create(:driver) }
+  let!(:rider) { create(:rider) }
+  let!(:trip) { create(:trip, driver_id: driver.id) }
+  let!(:review) { create(:review, trip_id: trip.id,
+                  rider_id: rider.id, driver_id: driver.id) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 16, 61 should be on its own line.